src/main/kotlin/kotliquery/Session.kt

```diff
@@ -131,13 +128,28 @@ open class Session(
         }
     }
 
+    fun updateWithKeys(query: Query): Long? {
```

Comment:
This implementation returns only one key. `updateAndReturnGeneratedKey` (not keys) is fine for me.